lib/twine-ui/src/styles/colours.ts

```diff
@@ -0,0 +1,51 @@
+export enum ColoursEnum {
```

Comment:
This was added a while ago, is this still consistent with (1) any updates to the design system and (2) the equivalent files in the dashboard?